src/wallet/wallet.cpp

```diff
@@ -4438,6 +4465,7 @@ util::Result<MigrationResult> MigrateLegacyToDescriptor(const std::string& walle
     // both before and after reloading. This ensures the set is complete even if one of the wallets
     // fails to reload.
     std::set<fs::path> wallet_dirs;
+    options.require_format = std::nullopt;
```

Comment:
It's used by the `reload_wallet` calls which capture it.

Reply:
Thanks, sorry for missing that. Worth an explaining comment like `"Reset require_format since reload_wallet will only be called on non-Berkeley DB sub-wallets from here on."` or something more correct?

Reply:
Added a comment.